src/NuGet.Clients/NuGet.CommandLine/Commands/InstallCommand.cs

```diff
@@ -199,7 +199,7 @@ private async Task PerformV2RestoreAsync(string packagesConfigFilePath, string i
                 cacheContext.NoCache = NoCache;
                 cacheContext.DirectDownload = DirectDownload;
 
-                var downloadContext = new PackageDownloadContext(cacheContext, installPath, DirectDownload);
+                var downloadContext = new PackageDownloadContext(cacheContext, Settings, installPath, DirectDownload);
```

Comment:
I'm not a fan of passing down an API such as Settings only to check 1 value (the revocation mode). 

I would personally evaluate the revocation mode and pass that only that information.